agentMET4FOF/agents.py

```diff
@@ -118,10 +175,15 @@ def log_info(self, message):
         """
         try:
             if self.log_mode:
-                super().log_info(message)
+                if self.backend == "osbrain":
+                    super().log_info(message)
+                elif self.backend == "mesa":
+                    message = '[%s] (%s): %s' % (datetime.datetime.utcnow(), self.name, message)
+                    print(message)
 
-        except Exception:
-                return -1
+        except Exception as e:
+            print(e)
+            return -1
```

Comment:
Referring to this part of the official Python documentation, we should consider changing this to something greater zero:

https://docs.python.org/2/library/sys.html#sys.exit.

Reply:
that's a good point, wasn't aware of it before..

Reply:
Resolved in 7ad9677